src/components/expression/expression-panel.js

```diff
@@ -224,16 +218,66 @@ genotet.ExpressionPanel.prototype.updateGenes = function(gene) {
       text: gene
     };
   });
-
   var profile = this.container.find('#profile select').empty();
-  this.selectProfiles_ = profile.select2({
-    data: genes,
-    multiple: true
-  });
-  this.container.find('#profile .select2-container').css({
-    width: '100%'
-  });
-  this.updateGeneSelections();
+
+  if (!genes.length) {
+    return;
+  }
+
+  $.fn.select2.amd.require(['select2/data/array', 'select2/utils'],
```

Comment:
I tried your codes above. In this function, an error said 'genotet.utils.CustomData.__super__ is undefined'. Are you sure CustomData can be defined out of amd function?
```js
/**
 * @param {!Element} $element
 * @param {!Object} options
 * @constructor
 */
genotet.utils.CustomData = function($element, options) {
  /**
   * @type {{
   *   __super__: function(?, ?)
   * }}
   */(genotet.utils.CustomData).__super__.constructor.call(
    this, $element, options);
};
```